assessments/projects/longhorn/self-assessment.md

```diff
@@ -0,0 +1,436 @@
+# Longhorn Self-assessment
+This is the Longhorn's Self-assessment documentation that we wrote after we began thinking about the security of the project, determining gaps in its security, and preparing security documentation for Longhorn's users. This project is currently in the CNCF Incubation phase preparing for Graduation.
+
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Threat Modelling](#Threat-Modelling)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+* [Action items](#Action-Items)
+
+## Metadata
+
+
+|   |  |
+| -- | -- |
+| Software | [A link to the Longhorn’s repository.](https://github.com/longhorn/longhorn) |
+| Security Provider | No |
+| Languages | Python, Shell, Mustache |
+| SBOM | The information is not readily available online and we have requested for the same. This is an action item for our subsequent week |
+| | |
+
+### Security links
+
+| Doc | url |
+| -- | -- |
+| Security file | https://longhorn.io/docs/1.5.3/advanced-resources/security/ |
```

Comment:
There is no content in this page. Can you please provide references to the secure development practices or other security content stated in this assessment?

Reply:
Hello, Ms. Ragashree Shekar. Thank you for the question. 
The URL points to the landing page of Security of Longhorn. While this page does not have any major information, it serves as an introduction/landing for security, and it is within the subpages that we can find more information on the Security.
The links to those sub-pages are also provided in the subsequent two rows of the table here. Hope that helps :) Thanks!

Reply:
If there are any questions, I will be happy to answer :) If there are none, may I go ahead and resole this conversation?

Reply:
As we are nearing the deadline, we are trying our best to not leave any unresolved questions and requests. Since there are no follow up questions for this thread, I am resolving this conversation. Please prompt us with any questions if you have any in the future and we will be happy to get back to you as soon as possible. Thanks!